examples/inverse/plot_evoked_ers_source_power.py

```diff
@@ -0,0 +1,136 @@
+
+The first is using a Dynamic Imaging of Coherent Sources (DICS) filter, more
+fully discussed in example: :ref:`ex-inverse-source-power`. The second uses an
+LCMV beamformer applied to active and baseline covariance matrices. Similarly
+the third approach computes minimum norm (MNE/dSPM) inverses and applies them
+to active and baseline covariance matrices.
+"""
+# Author: Luke Bloy <luke.bloy@gmail.com>
+#
+# License: BSD (3-clause)
+import os.path as op
+
+import numpy as np
+import mne
+from mne.cov import compute_covariance
+from mne.datasets import somato
+from mne.time_frequency import csd_morlet
+from mne.beamformer import (make_dics, apply_dics_csd, make_lcmv,
+                            apply_lcmv_cov)
+from mne.minimum_norm import (make_inverse_operator, apply_inverse_cov)
+
+print(__doc__)
+
+###############################################################################
+# Reading the raw data and creating epochs:
+data_path = somato.data_path()
+subject = '01'
+task = 'somato'
+raw_fname = op.join(data_path, 'sub-{}'.format(subject), 'meg',
+                    'sub-{}_task-{}_meg.fif'.format(subject, task))
+
+raw = mne.io.read_raw_fif(raw_fname)
+
+# We are interested in the beta band (12-30 Hz)
+raw.load_data().filter(12, 30)
+
+# The DICS beamformer currently only supports a single sensor type.
+# We'll use the gradiometers in this example.
+picks = mne.pick_types(raw.info, meg='grad', exclude='bads')
+
+# Read epochs
+events = mne.find_events(raw)
+epochs = mne.Epochs(raw, events, event_id=1, tmin=-1.5, tmax=2, picks=picks,
+                    preload=True)
+
+# Read forward operator and point to freesurfer subject directory
+fname_fwd = op.join(data_path, 'derivatives', 'sub-{}'.format(subject),
+                    'sub-{}_task-{}-fwd.fif'.format(subject, task))
+subjects_dir = op.join(data_path, 'derivatives', 'freesurfer', 'subjects')
+
+fwd = mne.read_forward_solution(fname_fwd)
+# fix the subject in fwd['src'] (it's mis-labeled in the dataset)
+for src in fwd['src']:
+    src['subject_his_id'] = subject
+
+###############################################################################
+# Compute covariances
+# -------------------
+# ERS activity starts at 0.5 seconds after stimulus onset.
+
+active_win = (0.5, 1.5)
+baseline_win = (-1, 0)
+
+baseline_cov = compute_covariance(epochs, tmin=baseline_win[0],
+                                  tmax=baseline_win[1], method='shrunk',
+                                  rank=None)
+active_cov = compute_covariance(epochs, tmin=active_win[0], tmax=active_win[1],
+                                method='shrunk', rank=None)
+
+# Weighted averaging is already in the addition of covariance objects.
+common_cov = baseline_cov + active_cov
+
+
+###############################################################################
+# Compute some source estimates
+# -----------------------------
+# Here we will use DICS, LCMV, and dSPM.
```

Comment:
Well, DICS can probably get away without it (short for "Dynamic imaging of coherent sources"), but it would be more explicit in contrast to the dSPM model of course.